README.md

```diff
@@ -68,10 +68,17 @@ ln -s ~/workspace/Stepup-Deploy ~/workspace/Stepup-VM/deploy
 
 Vagrant is used to create the app.stepup.example.com VM. Vagrant uses Ansible to provision to configure the networking in the VM and to do a yum update. This takes a while. If you have the resources, you could increase the memory from 2 to 3 or 4 GB by editing the ~/workspace/Stepup-VMVagrant file.
 
+Add the following to your hosts-file:
+
 ```
 192.168.66.3 app.stepup.example.com gateway.stepup.example.com selfservice.stepup.example.com ra.stepup.example.com tiqr.stepup.example.com tiqr.stepup.example.com middleware.stepup.example.com ks.stepup.example.com keyserver.stepup.example.com db.stepup.example.com ssp.stepup.example.com
 192.168.66.4 manage.stepup.example.com
 ```
+Start the creation of the Virtual machines by running
+```
+~/workspace/Stepup-Deploy 
+vagrant up
+```
```

Comment:
This block is certainly valuable, but maybe a pointer to go look at the `Development Stepup-VM guide` section of the readme would be better here?